components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2046,18 +2058,39 @@ private void parseOpenIDConnectConfig(OMElement oauthConfigElem) {
                                 .getText().trim();
             }
 
-            if (openIDConnectConfigElem.getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.ENCRYPTION_ALGORITHM)) != null) {
-                idTokenEncryptionAlgorithm =
-                        openIDConnectConfigElem.getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.ENCRYPTION_ALGORITHM))
+            if (openIDConnectConfigElem.getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.ID_TOKEN_ENCRYPTION_ALGORITHM)) != null) {
```

Comment:
Length seems to be exceeding 120 characters

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/960